public/components/integrations/components/setup_integration.tsx

```diff
@@ -4,97 +4,148 @@
  */
 
 import * as Eui from '@elastic/eui';
```

Comment:
Why aren't you just importing the components directly?

Reply:
Too many imports